packages/eas-cli/src/commands/branch/create.ts

```diff
@@ -16,6 +16,7 @@ import { getDisplayNameForProjectIdAsync } from '../../project/projectUtils';
 import { promptAsync } from '../../prompts';
 import { enableJsonOutput, printJsonOnlyOutput } from '../../utils/json';
 
+// NOTE(cedric): copied to src/branch/queries.ts to reuse in multiple commands
```

Comment:
can these methods be removed now that they've been factored out?

Reply:
Yeah, pending my open questions. I left a comment so I could do a quick search&destroy operation 😄

Reply:
Done!